### PR TITLE
Hopefully Unnoticeable Tool Refactor

### PR DESCRIPTION
**Pull request by Anewbe**
The playerside result should be basically nothing, hopefully.

- Replaces most of the `istype(thing, tool)` procs with newish `thing.is_tool()` procs. This will allow future items to act as a wrench, screwdriver, crowbar, or wirecutter. Not done (mostly) with welders, multitools, or cable coils, because they've got specific functionality that can't be replicated easily.
- Repaths wrenches, screwdrivers, crowbars, and wirecutters, to be part of an `obj/item/weapon/tool` subtype. Started as part of finding every istype proc, but I kind of like it, so it stuck.

TODO:
- [x] ~~Swap power tools to not be subtypes of normal tools.~~ Fail to make combitools work in a practical manner, continue to be perplexed by lists of datums.
- [x] Find and correct the places where I made `istype` into `is_tool` where I shouldn't have, ie. Welders, Multitools, and Cable Coils. Could use help finding them, etc.
- [x] Get mad because map merger things.

### CODE REVIEW

File: maps/submaps/surface_submaps/mountains/backup/crashed_ufo.dmm [hunk 1]
```diff
@@ -0,0 +1,112 @@
+"aa" = (/turf/template_noop,/area/template_noop)
```
Comment (Atermonera) [inline maps/submaps/surface_submaps/mountains/backup/crashed_ufo.dmm]: Backup map

File: maps/submaps/surface_submaps/mountains/backup/deadBeacon.dmm [hunk 2]
```diff
@@ -0,0 +1,62 @@
+"a" = (/turf/template_noop,/area/template_noop)
```
Comment (Atermonera) [inline maps/submaps/surface_submaps/mountains/backup/deadBeacon.dmm]: Backup map

File: maps/submaps/surface_submaps/mountains/backup/deadspy.dmm [hunk 3]
```diff
@@ -0,0 +1,32 @@
+"a" = (/obj/effect/decal/remains/xeno,/obj/item/clothing/mask/gas,/obj/item/clothing/suit/fire/firefighter,/turf/simulated/mineral/floor/ignore_mapgen,/area/submap/deadspy)
```
Comment (Atermonera) [inline maps/submaps/surface_submaps/mountains/backup/deadspy.dmm]: Backup map

File: maps/submaps/surface_submaps/mountains/backup/digsite.dmm [hunk 4]
```diff
@@ -0,0 +1,63 @@
+"a" = (/turf/template_noop,/area/template_noop)
```
Comment (Atermonera) [inline maps/submaps/surface_submaps/mountains/backup/digsite.dmm]: Backup map

File: maps/submaps/surface_submaps/mountains/backup/quarantineshuttle.dmm [hunk 5]
```diff
@@ -0,0 +1,146 @@
+"aa" = (/turf/template_noop,/area/template_noop)
```
Comment (Atermonera) [inline maps/submaps/surface_submaps/mountains/backup/quarantineshuttle.dmm]: Backup map

File: maps/submaps/surface_submaps/plains/backup/construction1.dmm [hunk 6]
```diff
@@ -0,0 +1,51 @@
+"a" = (/turf/template_noop,/area/template_noop)
```
Comment (Atermonera) [inline maps/submaps/surface_submaps/plains/backup/construction1.dmm]: Backup map

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 7]
```diff
@@ -767,7 +767,7 @@
 		else
 			dat += text("[obj]: \[<A HREF=?src=\ref[src];act=\ref[obj]>Activate</A> | <B>Deactivated</B>\]<BR>")
 */
-	src << browse(dat, "window=robotmod")
+	to_chat(src, browse(dat, "window=robotmod"))
```
[the remark this answers is not in the document]
Comment (Anewbe) [reply code/modules/mob/living/silicon/robot/robot.dm]: Last I checked it does not, don't actually know how that got replaced.

File: code/game/machinery/mass_driver.dm [hunk 8]
```diff
@@ -55,7 +55,7 @@
 			O_limit++
 			if(O_limit >= 20)
 				for(var/mob/M in hearers(src, null))
```
[the remark this answers is not in the document]
Comment (Anewbe) [reply code/game/machinery/mass_driver.dm]: If that's not code I directly touched, I will not be touching it in this particular PR. There's already enough in here, I don't want to play any more runtime Jenga than I have to.

**Pull request merged**